merkletree/str.go

```diff
@@ -21,10 +21,10 @@ type SignedTreeRoot struct {
 	PreviousEpoch   uint64
 	PreviousSTRHash []byte
 	Signature       []byte
-	Policies        Policies
+	Policies        *Policies
```

Comment:
This is a late night comment, so take it for what it's worth.  Is there any chance that changing this to a pointer is going to lead to the Policies in the STR being mutated?